rust/gui-client/src-tauri/src/main.rs

```diff
@@ -6,5 +6,11 @@
 mod client;
 
 fn main() -> anyhow::Result<()> {
+    // Mitigates a bug in Ubuntu 22.04
```

Comment:
Still required? What is the bug? Is it recorded somewhere?

Reply:
It's a bug with window decorations, documented it